crates/stc_ts_file_analyzer/src/analyzer/assign/mod.rs

```diff
@@ -1036,20 +1049,31 @@ impl Analyzer<'_, '_> {
                     _ => {}
                 }
             }
-            Type::EnumVariant(ref e @ EnumVariant { name: Some(..), .. }) => {
+            Type::EnumVariant(ref e @ EnumVariant { name: Some(name), .. }) => {
                 // Single-variant enums seem to be treated like a number.
                 //
                 // See typeArgumentInferenceWithObjectLiteral.ts
 
                 let items = self.find_type(&e.enum_name).context("failed to find an enum for assignment")?;
 
                 if let Some(items) = items {
-                    for e in items {
-                        match e.normalize() {
-                            Type::Enum(e) => {
-                                if e.members.len() == 1 {
+                    for t in items {
+                        match t.normalize() {
+                            Type::Enum(en) => {
+                                if en.clone().members.len() == 1 {
```

Comment:
Is clone necessary?

Reply:
Similar codes around it used to use `clone`.
If it succeeds with a `copy`, I will modify it like that.

Thank you for your feedback.

Reply:
didn't need it in this situation, sorry